make/autoconf/jdk-options.m4

```diff
@@ -477,6 +477,12 @@ AC_DEFUN_ONCE([JDKOPT_SETUP_UNDEFINED_BEHAVIOR_SANITIZER],
         LDFLAGS_JDKLIB="$LDFLAGS_JDKLIB $UBSAN_LDFLAGS"
         LDFLAGS_JDKEXE="$LDFLAGS_JDKEXE $UBSAN_LDFLAGS"
       ])
+  if test "x$UBSAN_ENABLED" = xfalse; then
```

Comment:
You don't have to explicitly declare variables empty; I don't think we normally do that.

Reply:
Mostly me being pedantic to prevent someone from accidentally referencing the variable without checking UBSAN_ENABLED.

Reply:
That would not matter anyway, they will be defined as empty strings in spec.gmk regardless. Oh well.